README.md

```diff
@@ -2,58 +2,112 @@
 
 # Running the Tests
 
+Each test suite is run in a separate Docker container using a temporary image built at test time.
+`conda` is used for dependency management. The two steps for each test suite are building and
+running the associated image.
+
 ## Install Prerequisites
 
 * [Docker](https://www.docker.com/get-started)
 
-## Build the Image & Run the Container
+## Build the Images
 
     $ cd test
-    $ make image
-    $ make run
-
-By default this will run the tests against the UAT environment. To run
-against a specific environment:
-
-    $ make run environment=prod
-
-Valid environment values are: sbx, sit, uat, prod.
-
-# Notebook Development
+    $ make images
 
-**Note** - this section applies to the contents of the `test` directory
+`make -j images` can be used to make the images in parallel (faster), although this may lead to
+Docker Desktop instabilities
 
-These prerequisites and steps are only needed if you want to do local
-development on the project. 
+## Create Terraform Autovars File
+In the `terraform` directory create a file called `key.auto.tfvars` and
+add a single line indicating the name of the ssh public key file that
+should be used for the EC2 instance that runs the notebooks.
 
-## Prerequisites
+This file name is the name of the S3 file created in the Harmony ssh key bucket as described in the Harmony project README.md.
 
-* [pyenv](https://github.com/pyenv/pyenv)
-* [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv)
-* [poetry](https://python-poetry.org/)
+Example:
+```
+key_name = "harmony-sit-my-key-name"
+```
 
-## Install Python 3.8 (if needed)
+## Run the notebooks
 
-    $ pyenv install 3.8.5
+    $ cd test
+    $ export HARMONY_HOST_URL=<url of Harmony in the target environment>
+    $ ./run_notebooks.sh
 
-## Install dependencies
+Outputs will be in the `output` directory. 
+`HARMONY_HOST_URL` for SIT would be `https://harmony.sit.earthdata.nasa.gov`
 
-    $ pyenv virtualenv 3.8.5 harmony-rt
-    $ pyenv local harmony-rt
-    $ pyenv activate harmony-rt
-    $ poetry install
-    $ pyenv rehash
+# Running the Tests in AWS
+First create a `.env` file in the top level directory by copying in the `dot_env` file and filling
+in the proper values. Then execute the following.
 
-## Run the notebooks
+    $ cd script
+    $ export HARMONY_ENVIRONMENT=<uat|sit|sandbox|prod>
+    $ ./test.sh
 
-    $ ./run_notebooks harmony_host_url=<url of Harmony in the target environment>
-    
-e.g., 
-    
-    $ ./run_notebooks harmony_host_url="https://harmony.sit.earthdata.nasa.gov"
+Output will be in the bucket specified with the `REGRESSION_TEST_OUTPUT_BUCKET` environment 
```

Comment:
1. From the README > First create a `.env` file in the top level directory by copying in the `dot_env` file and filling
in the proper values.
The dot_env file example has this var.
2. The output does not go to the local output directory when run in AWS (what this particular section is describing). You may have had output from a local run still in there.

Reply:
Some wires got crossed; I think I was looking at an outdated version of the README somehow.  Re-reviewing, because there are other things in here that I hadn't seen.

Reply:
Actually I might be wrong about the output...